astroid/test_utils.py

```diff
@@ -46,8 +47,18 @@ def _extract_expressions(node):
             child = getattr(node.parent, name)
             if isinstance(child, (list, tuple)):
                 for idx, compound_child in enumerate(child):
-                    if compound_child is node:
+
+                    # Can't find a cleaner way to do this.
```

Comment:
This comment could be expanded to clarify what this code is doing.  (I'm not clear on its purpose.)